toolchain/check/import_cpp.cpp

```diff
@@ -427,6 +444,45 @@ static auto GetReturnType(Context& context, SemIR::LocId loc_id,
   return param_pattern_id;
 }
 
+namespace {
+// Represents the parameter patterns block id, the return slot pattern id and
+// the call parameters block id for a function declaration.
+struct FunctionParamsInsts {
+  SemIR::InstBlockId param_patterns_id;
+  SemIR::InstId return_slot_pattern_id;
+  SemIR::InstBlockId call_params_id;
+};
+}  // namespace
+
+// Creates a block containing the parameter pattern instructions for the
+// explicit parameters, a parameter pattern instruction for the return type and
+// a block containing the call parameters of the function. Emits a callee
+// pattern-match for the explicit parameter patterns and the return slot pattern
+// to create the Call parameters instructions block. Currently the implicit
+// parameter patterns are not taken into account. Returns the parameter patterns
+// block id, the return slot pattern id, and the call parameters block id.
+// Returns `std::nullopt` if the function declaration has an unsupported
+// parameter type.
+static auto CreateFunctionParamsInsts(Context& context, SemIR::LocId loc_id,
+                                      const clang::FunctionDecl* clang_decl)
+    -> std::optional<FunctionParamsInsts> {
+  auto param_patterns_id =
+      MakeParamPatternsBlockId(context, loc_id, *clang_decl);
+  if (!param_patterns_id.has_value()) {
+    return std::nullopt;
+  }
+  auto return_slot_pattern_id = GetReturnType(context, loc_id, clang_decl);
+  if (SemIR::ErrorInst::InstId == return_slot_pattern_id) {
+    return std::nullopt;
+  }
+  // TODO: Add support for implicit parameters.
+  auto call_params_id =
+      CalleePatternMatch(context, SemIR::InstBlockId::None, param_patterns_id,
+                         return_slot_pattern_id);
+  return FunctionParamsInsts{param_patterns_id, return_slot_pattern_id,
+                             call_params_id};
```

Comment:
Our style guide prefers two things here:
1. Avoid writing the type name if it's not necessary (it's inferred from the return type here)
2. use designated initializers

```suggestion
  return {.param_patterns_id = param_patterns_id,
      .return_slot_pattern_id = return_slot_pattern_id,
      .call_params_id = call_params_id};
```

Reply:
Done, thanks.